apps/native/app/src/ui/lib/tableview/tableview-cell.tsx

```diff
@@ -40,12 +41,12 @@ interface TableViewCellProps {
 
 const Cell = styled.View<{ border: boolean; disabled: boolean }>`
   flex-direction: row;
-  min-height: 71px;
+  min-height: 56px;
```

Comment:
Will this height break some other components that are using this? Just wondering

Reply:
No I checked in all places that this component is used 👌 All good